### PR TITLE
Matched EnMb and EnDoor

### PR DESCRIPTION
**Pull request by petrie911**
More fully matched actors. Also z_sram was missing a comma which is now back.

### CODE REVIEW

File: src/overlays/actors/ovl_En_Door/z_en_door.c [hunk 1]
```diff
@@ -211,9 +207,11 @@ void EnDoor_Idle(EnDoor* this, GlobalContext* globalCtx) {
             }
             if (ABS(phi_v0) < 0x3000) {
                 if (this->lockTimer != 0) {
-                    numKeys = gSaveContext.inventory.dungeonKeys[gSaveContext.mapIndex];
-                    if (numKeys <= 0) {
-                        PLAYER->naviTextId = -0x203;
+                    // numKeys = ;
+                    if (gSaveContext.inventory.dungeonKeys[gSaveContext.mapIndex] <= 0) {
```
Comment (fig02) [inline src/overlays/actors/ovl_En_Door/z_en_door.c:211]: remove comment then its good

File: src/overlays/actors/ovl_En_Door/z_en_door.c [hunk 2]
```diff
@@ -211,9 +207,11 @@ void EnDoor_Idle(EnDoor* this, GlobalContext* globalCtx) {
             }
             if (ABS(phi_v0) < 0x3000) {
                 if (this->lockTimer != 0) {
-                    numKeys = gSaveContext.inventory.dungeonKeys[gSaveContext.mapIndex];
-                    if (numKeys <= 0) {
-                        PLAYER->naviTextId = -0x203;
+                    // numKeys = ;
```
Comment (petrie911) [inline src/overlays/actors/ovl_En_Door/z_en_door.c:210]:
```suggestion
```

File: src/overlays/actors/ovl_En_Door/z_en_door.c (hunk 1 above)
Comment (petrie911) [reply src/overlays/actors/ovl_En_Door/z_en_door.c]: should work now.

**Review by fig02 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Mb/z_en_mb.c [hunk 3]
```diff
@@ -1007,23 +1001,20 @@ void func_80AA87D8(EnMb* this, GlobalContext* globalCtx) {
...
+                this->soundTimer = Math_Rand_S16Offset(15, 40);
             } else {
                 func_80AA6830(this);
             }
         }
     }
     if (currentFrame != (s32)this->skelAnime.animCurrentFrame) {
-        if (!((sp48 >= 2) || (((s32)playbackSpeedABS + (s32)this->skelAnime.animCurrentFrame) <= 0)) ||
+        if (((sp48 <= 1) && ((s32)playbackSpeedABS + currentFrame >= 1)) ||
             ((sp48 < 21) && (((s32)playbackSpeedABS + currentFrame >= 20)))) {
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Mb/z_en_mb.c:1012]: Very minor but we might as well be consistent
```suggestion
            ((sp48 <= 20) && (((s32)playbackSpeedABS + currentFrame >= 20)))) {
```

Comment (petrie911) [reply src/overlays/actors/ovl_En_Mb/z_en_mb.c:1012]: Huh, I thought I did that.

**Review by Roman971 — APPROVED** [review]

**Pull request merged**